app/assets/javascripts/pageflow/timeline_page/components/timeline_item_caption.jsx

```diff
@@ -0,0 +1,17 @@
+(function() {
+  pageflow.timelinePage.TimelineItemCaption = class extends React.Component {
+    render() {
+      var tagline = this.props.tagline;
+      var caption = this.props.caption;
+
+      if(!(tagline || caption)) {
+        return null;
+      } else return(
+        <div className="timeline_item-caption">
+          <span className="timeline_item-tagline">{tagline}</span>
+          {caption}
+        </div>
+      );
+    }
```

Comment:
If Hound was around here, he should have said https://eslint.org/docs/rules/no-negated-condition ;)

Also, since I noticed it in a few other places, even if there is no linting rule for that, the rest of the Pageflow code base uses a space after keywords like `if` and `return`.